CMakeLists.txt

```diff
@@ -4,20 +4,12 @@ project(demonstrator)
 # Print the value of CMAKE_SOURCE_DIR for debugging
 message(STATUS "CMAKE_SOURCE_DIR: ${CMAKE_SOURCE_DIR}")
 
-include(${CMAKE_BINARY_DIR}/conan_toolchain.cmake)
-
 add_executable(demonstrator src/demonstrator.c)
-find_package(module_a REQUIRED)
-find_package(module_b REQUIRED)
+find_package(module_a CONFIG REQUIRED)
```

Comment:
better be explicit that CONFIG is required

Reply:
what does CONFIG mean in this context?

Reply:
to use the CMake modern ``xxxx-config.cmake`` files for dependencies (called "config" files), instead of the legacy ``Findxxxx.cmake`` called "modules"